cmd/readvar/src/lib.rs

```diff
@@ -65,9 +65,9 @@ fn readvar_dump(
     let mut buf: Vec<u8> = vec![];
```

Comment:
General comment about the commit message. I think it would be helpful to the future readers if the commit message also explained why the change is being made for halt -> op_start and run -> op_done.